### PR TITLE
fixes instructions for QGIS install in dockerfile

### PR DESCRIPTION
**Pull request by haubourg**
Just an update of the QGIS gpg key

### CODE REVIEW

**Comment by elpaso** [comment]
Maybe worths adding a note/warning about the possibility that the key may change in the future. I doubt we will always remember to update it.

**Comment by DelazJ** [comment]
Agreed. And a link to the website page where this info can be found

**Comment by haubourg** [comment]
> Maybe worths adding a note/warning about the possibility that the key may change in the future. I doubt we will always remember to update it.

totally, I'm on it

**Comment by haubourg** [comment]
@elpaso @DelazJ I pushed some comments there

File: docs/server_manual/containerized_deployment.rst [hunk 1]
```diff
@@ -20,6 +20,10 @@ simple (simple Docker images) to sophisticated (Kubernetes and so on).
    which can be retrieved as sources (Dockerfile and resources) to
    build or already built from registries (private or public).
 
+.. note:: QGIS Debian-Ubuntu package downloads need a valid gpg authentication key.  
+   Please refer to the `installation pages https://www.qgis.org/fr/site/forusers/alldownloads.html#debian-ubuntu`_
```
Comment (DelazJ) [inline docs/server_manual/containerized_deployment.rst:24]:
```suggestion
   Please refer to the `installation pages <https://www.qgis.org/fr/site/forusers/alldownloads.html#debian-ubuntu>`_ 
```

**Comment by haubourg** [comment]
Thanks @DelazJ !

**Pull request merged**